### ISSUE TITLE
Hangul 한글 characters classification by Quantum Machine Learning

### ISSUE THREAD

**Issue by kifumi** [qiskit-community/qiskit-hackathon-korea-22#5]
# Abstract
Construct Hangul MNIST classifier using Qiskit Quantum Machine Learning.

# Description 
We have MNIST classifier made by 1) [Quantum Support Vector Machine](https://learn.qiskit.org/content/summer-school/2021/resources/lab-notebooks/lab-3.ipynb) - Qiskit Global Summer School 2021, 2)[Torch Connector and Hybrid QNNs](https://qiskit.org/documentation/machine-learning/tutorials/05_torch_connector.html#Part-2:-MNIST-Classification,-Hybrid-QNNs). Let’s try to make Hangul characters classifier using Qiskit and adjust the method to improve the accuracy. 

We can use [Hangul Datasets](https://github.com/callee2006/HangulDB) which Sophy found for us or [Handwritten Hangul Characters](https://www.kaggle.com/wayperwayp/hangulkorean-characters).

# Members

 - @FmnFlower
 - @kjchoiquantum 
 - @Jason-L1 
 - @jin-castle
 - @jhlee29
 - @githubhandle 
 - @githubhandle - Slack: `@slackhandle` email: `example@example.com`
 - Qiskit Coach: @kifumi  @attp @String137 @tula3and 

# Deliverable

# GitHub repo
https://github.com/jhlee29/quantum-meets-hangul

**Comment by attp** [comment]
I'm interested in mentoring this project as well!

**Comment by kifumi** [comment]
@attp Anna, thank you!! We got a super mentor!

**Comment by String137** [comment]
I'm also interested in mentoring this project! Can I join as a mentor?

**Comment by 0sophy1** [comment]
> I'm also interested in mentoring this project! Can I join as a mentor?

@kifumi  Hojun is one of our mentor, this would be a good chance for Hojun learn more about QML from awesome senior mentors

**Comment by FmnFlower** [comment]
I'm interested in this issue

**Comment by kjchoiquantum** [comment]
I am really interested in this project. Can I join as a member?

**Comment by Jason-L1** [comment]
I am also interested in this project. Can I join as a member?

**Comment by jin-castle** [comment]
I am really interested in this project. Can I join as a member?

**Comment by ktasha45** [comment]
Hi. This is Dongyoung, who I told you earlier on gather that I will be contacting you.
I am reading (https://qiskit.org/textbook/ch-machine-learning/machine-learning-qiskit-pytorch.html) now. However, it seems that it will take some time to understand the code of pytorch because I have only used tensorflow before. Since you said you use SVM, we are going to use CNN. But honestly, I don't know if that's possible. Because like I said, I've only used TensorFlow.

As I said our team is already 4 people, so it would be difficult for all of our teams to participate in your team.
So, as you said, I propose to split the team. I thought about dividing the team and trying to solve the same topic in different ways.

But in order to do that, our team first needs to understand the open code shown in the link.
So, let's see if our team can understand that code, and if we can process the code at our level, we'll let you know. I'll tell you by tomorrow afternoon at the latest.
thank you. If I'm wrong, or if you don't want to collaborate with us, don't hesitate to tell me.

**Comment by 0sophy1** [comment]
Do you want more members? if not, I'll remove member wanted label

**Comment by jhlee29** [comment]
Hello! I would like to join this project. Is there still a room in this group?

**Comment by kifumi** [comment]
@0sophy1 How about dividing the team into two if the members exceed the limit?

**Comment by 0sophy1** [comment]
If you are OK with that I'm definitely follow your opinion :) Thank you so much!

**Comment by tula3and** [comment]
If there is room for mentors, I want to join this project! :)

**Comment by kifumi** [comment]
@tula3and Maybe this team will split to two teams, so the mentors should also split to the two teams.  So, please join us!

**Comment by tula3and** [comment]
@kifumi I'd love to join. Thanks!

**Comment by ktasha45** [comment]
As I said, our team is likely to proceed with the project using the Hangul dataset and QCNN.
However, the dataset may change.

**Comment by kifumi** [comment]
@ktasha45 Yes, I think the dataset is key for this project.. Could you please open new issue? And, please tell your members to leave the comment on the new issue.

**Comment by 0sophy1** [comment]
Wow.. kifumi congraturation! such a mega project!

**Comment by ktasha45** [comment]
> @ktasha45 Yes, I think the dataset is key for this project.. Could you please open new issue? And, please tell your members to leave the comment on the new issue.

I'll do that. as soon as possible.

**Comment by kifumi** [comment]
@FmnFlower
@kjchoiquantum
@Jason-L1
@jin-castle
Hi everyone, could you please tell me your slack name? Or, please join the Room on veertly! Veertly에서 왼쪽 메뉴에 Rooms를 클릭하면 이슈에 맞는 방이 만들어져 있어요! 찾기 힘드시면 알려주세요~!
